src/Suggest.purs

```diff
@@ -32,28 +32,23 @@ type Replacement =
   , replacement :: String
 }
 
-type SEff eff h = (console :: CONSOLE, exception :: EXCEPTION, fs :: FS, st :: ST h | eff)
-
--- This alias avoids psc bug which should be fixed in 9.0
-type PsaAnnotedErrors = Array PsaAnnotedError
```

Comment:
I wonder if the typo was required to fix the bug :D

Reply:
:D